### PR TITLE
Covered Ed Faulkner's presentation at EmberATX (issue 104)

### PR DESCRIPTION
**Pull request by ijlee2**

### CODE REVIEW

File: source/2019-06-28-the-ember-times-issue-104.md [hunk 1]
```diff
@@ -31,12 +32,17 @@ Help out in the Prettier Glimmer Quest Issue 🎀✨,
...
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+You might be wondering next, how do I start using [Embroider](https://github.com/embroider-build/embroider) in my app? How exactly do Embroider options affect my build? 🙋‍♂️🙋‍♀️
+
+At EmberATX, [Edward Faulkner](https://github.com/ef4) showed how you can build the familiar [Super Rentals app](https://guides.emberjs.com/release/tutorial/ember-cli/) using Embroider. He also taught important [core and compatibility options](https://github.com/embroider-build/embroider#options) and explained how they affect the first two build stages.
+
+We encourage you to [learn more about Embroider today](https://www.youtube.com/watch?v=2R1vS7_sLTM)! If you are short on time, feel free to watch these highlights:
+
+- *08:20 (24 min)* How to build Super Rentals 🏠
```
Comment (jayjayjpg) [inline source/2019-06-28-the-ember-times-issue-104.md]:
```suggestion
- *[08:20](https://www.youtube.com/watch?v=2R1vS7_sLTM&feature=youtu.be&t=500)(24 min)* How to build Super Rentals 🏠
```

File: source/2019-06-28-the-ember-times-issue-104.md [hunk 2]
```diff
@@ -31,12 +32,17 @@ Help out in the Prettier Glimmer Quest Issue 🎀✨,
...
+- *47:25 (25 min)* How the options affect first stage 1️⃣
```
Comment (jayjayjpg) [inline source/2019-06-28-the-ember-times-issue-104.md]:
```suggestion
- *[47:25](https://www.youtube.com/watch?v=2R1vS7_sLTM&feature=youtu.be&t=2845) (25 min)* How the options affect first stage 1️⃣
```

File: source/2019-06-28-the-ember-times-issue-104.md [hunk 3]
```diff
@@ -31,12 +32,17 @@ Help out in the Prettier Glimmer Quest Issue 🎀✨,
...
+- *1:02:25 (15 min)* How the options affect second stage 2️⃣
```
Comment (jayjayjpg) [inline source/2019-06-28-the-ember-times-issue-104.md]:
```suggestion
- *[1:02:25](https://www.youtube.com/watch?v=2R1vS7_sLTM&feature=youtu.be&t=3745) (15 min)* How the options affect second stage 2️⃣
```

**Review by jayjayjpg — APPROVED** [review]
This reads great! Does it make sense to add youtube time stamps to the separate sections of the video? Feel free to apply the suggestions if you find it useful and thank you for another :sparkles: writeup this week!

**Comment by ijlee2** [comment]
@jessica-jordan Much thanks for your review and suggestions! Linking timestamps is a great idea!

**Pull request merged**